testbed/tests/app/test_mobile.py

```diff
@@ -17,12 +18,51 @@ async def test_show_hide_cursor(app):
     app.hide_cursor()
 
 
-async def test_full_screen(app):
-    """Window can be made full screen"""
-    # Invoke the methods to verify the endpoints exist. However, they're no-ops,
-    # so there's nothing to test.
-    app.set_full_screen(app.current_window)
-    app.exit_full_screen()
+async def test_presentation_mode(app, main_window, main_window_probe):
+    """The app can enter into presentation mode"""
+    if not main_window_probe.supports_presentation:
+        pytest.xfail("This backend doesn't support presentation window state.")
+
+    assert not app.in_presentation_mode
+    assert main_window.state != WindowState.PRESENTATION
+
+    # Enter presentation mode with main window via the app
+    app.enter_presentation_mode([main_window])
```

Comment:
The backend doesn't need to verify dict vs list mode operation. That's validated by the core tests. The testbed only needs to validate that the *backend* states are followed.